sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -2974,9 +2974,9 @@ class Analyzer(
    */
   object ResolveWindowFrame extends Rule[LogicalPlan] {
     def apply(plan: LogicalPlan): LogicalPlan = plan resolveExpressions {
-      case WindowExpression(wf: WindowFunction, WindowSpecDefinition(_, _, f: SpecifiedWindowFrame))
-          if wf.frame != UnspecifiedFrame && wf.frame != f =>
-        failAnalysis(s"Window Frame $f must match the required frame ${wf.frame}")
+      case WindowExpression(owf: OffsetWindowFunction,
+        WindowSpecDefinition(_, _, _: SpecifiedWindowFrame)) =>
```

Comment:
`OffsetWindowFunction` can use `UnspecifiedFrame` now?

Reply:
I got it.